apps/studio/src/lib/db/clients/postgresql.ts

```diff
@@ -288,10 +290,25 @@ export async function listTablePartitions(conn: HasPool, tableName: string) {
     WHERE base_tb.oid = ?::regclass
   `, [tableName]).toQuery();
```

Comment:
@not-night-but this is where we need to also filter by schema. This only filters by tableName